packages/native-stack/src/types.tsx

```diff
@@ -37,6 +36,15 @@ export type NativeStackNavigationEventMap = {
    * Event which fires when a swipe back is canceled on iOS.
    */
   gestureCancel: { data: undefined };
+  /**
+   * Event which fires when screen is in sheet presentation & it's detent changes.
+   *
+   * In payload it caries two fields:
+   *
+   * * index - current detent index in the `sheetAllowedDetents` array,
+   * * isStable - on Android `false` value means that the user is dragging the sheet or it is settling; on iOS it is always `true`.
+   */
+  sheetDetentChange: { data: { index: number; isStable: boolean } };
```

Comment:
The `isStable` naming seems confusing. If this is about dragging, then wouldn't `dragging: boolean` be better?

Reply:
+1

Reply:
When I'm thinking of it right now - `dragging` might be misleading here, e.g. if someone wanted to dim sheet's contents on drag gesture. `isStable` is also `false`, when user is no longer dragging the sheet, but it is still settling (animating) to target detent.

Reply:
@kkafar ok, lets change the property name to `stable`. we're not using `is` prefix in other places in such properties. e.g. stack has `closing`